doc/source/user_guide/math.rst

```diff
@@ -1,62 +1,30 @@
-an entirely new language because APDL Math has been written in a similar manner
-as the ``numpy`` and ``scipy`` libraries. For example, here is a comparison between
-the NumPy and SciPy linear algebra solvers and the Ansys MAPDL Math solver:
-
-.. table:: ``numpy`` vs PyMAPDL Math Implementation
-
-   +--------------------------------------------+-----------------------------------+
-   | ``numpy`` and ``scipy``                    | ``ansys.mapdl.math``              |
-   +============================================+===================================+
-   | .. code:: python                           | .. code:: python                  |
-   |                                            |                                   |
-   |   k_py = k + sparse.triu(k, 1).T           |   k = mm.matrix(k_py, triu=True)  |
-   |   m_py = m + sparse.triu(m, 1).T           |   m = mm.matrix(m_py, triu=True)  |
-   |   n = 10                                   |   n = 10                          |
-   |   ev = linalg.eigsh(k_py, k=neqv, M=m_py)  |   ev = mm.eigs(n, k, m)           |
-   |                                            |                                   |
-   +--------------------------------------------+-----------------------------------+
-
-What follows is a basic example and a detailed description of the
-PyMAPDL Math API. For additional PyMAPDL Math examples, see
-:ref:`ref_apdl_math_examples`.
-
-
-MAPDL matrix example
-~~~~~~~~~~~~~~~~~~~~
-This example demonstrates how to send an MAPDL Math matrix from MAPDL
-to Python and then send it back to be solved. While this example runs the
-:func:`MapdlMath.eigs() <ansys.mapdl.core.math.MapdlMath.eigs>` method on mass
-and stiffness matrices generated from MAPDL, you could instead use
-mass and stiffness matrices generated from an external FEM tool or
-even modify the mass and stiffness matrices within Python.
-
-First, solve the first 10 modes of a ``1 x 1 x 1`` steel meter cube
-in MAPDL.
+PyAnsys Math overview
+=====================
+`PyAnsys Math <pyansys_math_>`_ provides the ability to access and manipulate
+the large sparse matrices and solve a variety of eigenproblems in a similar
+manner to the popular `numpy <numpy_docs_>`_ and `scipy <scipy_docs_>`_ libraries.
+
+
+PyMAPDL and PyAnsys Math
+~~~~~~~~~~~~~~~~~~~~~~~~
+This example demonstrates how to take advantage of the `ansys-math-core` package
+with PyMAPDL.
+
+.. This example demonstrates how to send an MAPDL Math matrix from MAPDL
+.. to Python and then send it back to be solved. While this example runs the
+.. :func:`mm.eigs() <ansys.math.core.math.AnsMath.eigs>` method on mass
+.. and stiffness matrices generated from MAPDL, you could instead use
+.. mass and stiffness matrices generated from an external FEM tool or
+.. even modify the mass and stiffness matrices within Python.
+
+.. First, solve the first 10 modes of a ``1 x 1 x 1`` steel meter cube
+.. in MAPDL.
 
```

Comment:
Is this intended to be a comment?

Reply:
Looks like it is just coming from the old description, please uncomment and adapt, or remove. As you prefer.